### ISSUE TITLE
How can I install LSpec?

### ISSUE THREAD

**Issue by chanind** [lurk-lab/LSpec#41]
Apologies if this is obvious, but how can I install this library? Is there a lake or elan command that installs it? I added the following line to my lakefile.lean and ran `lake update` and it seems to sort of work:

```
require lspec from git
  "https://github.com/lurk-lab/LSpec.git"
```

Is this correct? What's the proper way to install LSpec library in a project?

**Comment by arthurpaulino** [comment]
Yeah, as far as I know, that's the current way of adding Lean libs to your project.

**Comment by chanind** [comment]
Ah ok, thanks - amazingly fast reply! Is there anything else I need to add to lakefile to get this to work? I see the following in the README:

```
The lspec binary searches for binary executables defined in lakefile.lean whose module name starts with "Tests". Then it builds and runs each of them.
```

If I have my project set up like:

```
├── MyProject
│   └── thing.lean
├── lakefile.lean
```

does this means I should be setting up the tests like the following?

```
├── MyProject
│   └── thing.lean
├── Tests
│   └── test_thing.lean
├── lakefile.lean
```

If so, what do I need to add to the lakefile to make this work with `lake exe lspec`?

Apologies for what I'm sure is a very basic question!

**Comment by arthurpaulino** [comment]
You also need to add the modules containing tests in your `lakefile.lean`. Example:
https://github.com/lurk-lab/YatimaStdLib.lean/blob/main/lakefile.lean#L68-L76

I think that should be enough to enable `lake exe lspec`. If not, let me know. It's been a while since I've done work in Lean 4.

Make sure to check how the tests are defined in `YatimaStdLib` so you can understand how to speak "LSpec's language".

**Comment by chanind** [comment]
Seeing an example project really helps make things clear! Thank you so much!
